app/assets/javascripts/hubstats/deploys.js

```diff
@@ -0,0 +1,78 @@
+// Place all the behaviors and hooks related to the matching controller here.
+// All this logic will automatically be available in application.js.
+
+$(document).ready(function() {
+  queryParameters = getUrlVars();
+  setDefaults(queryParameters);
+  activeLabels(queryParameters);
+  //initLabels(queryParameters)
+  changeColors();
+
+  $("#state-group > .btn").on("click", function(){
+    updateQueryStringParameter(queryParameters,"state",$(this).attr('id'));
+  });
+
+  $("#sort-group > .btn").on("click", function(){
+    updateQueryStringParameter(queryParameters,"order",$(this).attr('id'));
+  });
+
+  $("#group-by").on("change", function(){
+    updateQueryStringParameter(queryParameters,"group",$(this)[0].value);
+  });
+
+  $("#repos").change(function() {
+    var ids = $("#repos").val();
+    updateQueryStringParameter(queryParameters,"repos",ids);
+  });
+  
+  $("#users").change(function() {
+    var ids = $("#users").val();
+    updateQueryStringParameter(queryParameters,"users",ids);
+  });
+});
+
+function updateQueryStringParameter(queryParameters, key, value) {
```

Comment:
Technically, it doesn't need to be.  It could be in any file.  But its confusing for developers to have code in a file called 'pull_requests' that is also used else where.  For the sake of human readability and organization, we should move the shared code to a file that isn't dedicated to one resource/controller by its name.